charts/golang/templates/tlscontext.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if and (.Capabilities.APIVersions.Has "getambassador.io/v2/TLSContext") .Values.ingress.enabled .Values.ingress.tls.enabled (eq .Values.ingress.class "ambassador")}}
```

Comment:
Added `.Capabilities.APIVersions.Has "getambassador.io/v2/TLSContext"` because `TLSContext` is otherwise missing for chart-testing to pass successfully.